sos/cleaner/obfuscation_archive.py

```diff
@@ -73,7 +73,9 @@ def _load_skip_list(self):
             'sys/kernel/debug',
             'sys/module',
             'var/log/.*dnf.*',
-            '.*.tar.*',  # TODO: support archive unpacking
+            '.*\.tar$',  # TODO: support archive unpacking
+            # Be explicit with these tar matches to avoid matching commands
+            '.*\.tar\.xz',
```

Comment:
What about `.*\.tar\.[g,x]z` ?

Reply:
Ignore this, I just noticed line 79.

Reply:
Yeah, I thought about doing that too but I'm err'ing on the side of caution of not wanting another surprise like this down the road.